packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -12,54 +12,86 @@ Policies are evaluated when a request is made. Rules in the policies define perm
 
 There are many possibilities for policy use:
 
-* Create authorization rules based on complex logic using conditions
-* Determine extra levels of authentication that must be performed before a specific Okta application can be accessed
-* Maintain a list of allowed users and deny access based on multiple conditions
-* Change the returned scopes of the access token and add claims to it and to the ID token using [inline hooks](/docs/concepts/inline-hooks/)
-* Notify other systems through an API when a sign in happens in real-time
+* Create authorization rules based on complex logic using conditions.
+* Determine extra levels of authentication that must be performed before a user accesses a specific Okta application.
+* Maintain a list of allowed users and deny access based on multiple conditions.
+* Change the returned scopes of the access token and add claims to it and to the ID token using [inline hooks](/docs/concepts/inline-hooks/).
+* Notify other systems in real-time through an API when a user signs in.
 * Use the scopes of a token to look up user information in an external database or API, then add that data to the user's profile object.
 
+### Default policies
+
+A default policy is automatically created for each type of policy. This ensures that there is always a policy to apply to a user in all situations. Default policies are required, and you can't delete them. They are always the last policy in the priority order and any added policies of the same type have higher priority. Default policies also always have one default rule that you can’t delete, and that rule is always the last rule in the priority order. When you add rules to the default policy, they have a higher priority than the default rule. The `system` attribute is set to `TRUE` on system policies, which flags those policies that you can’t delete.
```

Comment:
What is a system policy? It feels implied here, but it's the only mention on the page. Maybe we could link somewhere?